registry/npm.js

```diff
@@ -4,13 +4,20 @@ class NpmRegistry {
   constructor () {
     this.constants = {
       registry: 'https://registry.npmjs.org/',
+      registryNerf: '//registry.npmjs.org/',
       readWrite: 'read-write',
       write: 'write'
     }
   }
 
   async getUsername ({ readOnlyToken }) {
-    const opts = { registry: this.constants.registry, token: readOnlyToken }
+    const opts = {
+      registry: this.constants.registry,
+      token: readOnlyToken,
+      forceAuth: {
+        _authToken: readOnlyToken
+      }
```

Comment:
this stops our API from ever deferring to some local npm config. my API tests wouldn't pass due to some BS here. now it always uses the auth token that we give it.